crates/sui-types/src/temporary_store.rs

```diff
@@ -459,6 +459,12 @@ impl<S> TemporaryStore<S> {
         status: ExecutionStatus,
         gas_object_ref: ObjectRef,
     ) -> (InnerTemporaryStore, TransactionEffects) {
+        let mutable_input_versions = self
+            .mutable_input_refs
+            .iter()
+            .map(|(id, v, _)| (*id, *v))
```

Comment:
Note to self (or others if you already know): Should I be including the gas object in this vector explicitly, or is it done for me?

Reply:
I believe it is already present, but cc @lxfind